torchtitan/models/llama3/model/model.py

```diff
@@ -191,9 +194,12 @@ def __init__(self, model_args: TransformerModelArgs):
             model_args.n_heads * self.head_dim, model_args.dim, bias=False
         )
 
-        self.use_flex_attn = model_args.use_flex_attn
+        self.use_flex_attn = model_args.attention_type == "flex"
+        self.use_varlen_attn = model_args.attention_type == "varlen"
```

Comment:
This doesn't seem to be required. You can just use `match model_args.attention_type` as you have done in the get_attention_mask().